### ISSUE TITLE
Test Help Page Interactive Elements for Accessibility

### ISSUE THREAD

**Issue by bpmcneilly** [internetarchive/openlibrary#4907]
# Page to Test:

Help Page - https://openlibrary.org/help/faq/about

# Steps

Steps for this task are outlined in #4897
 
# Expectations

Test the specified page according to the steps outlined in the parent issue. Create individual bugs with steps to reproduce for any found violations on this page. Reference those bugs on this tracking issue.

Once the page has been tested & all bugs are found, close this issue.

**Comment by ArunTeltia** [comment]
I tested every element in the https://openlibrary.org/help/faq/about page ,every element is keyboard accessible 
Tested by going from top to bottom and bottom to top.

There is one problem with the keyboard accessibility that I can think is the radio buttons are not accessible by keyboard this is the same issue over and over again, 
In the /help/faq/about page when worked on the history of the page 
The link to the histoy : https://openlibrary.org/help/faq/about?v=1&m=history

Here radio buttons are not accessible

**Comment by bpmcneilly** [comment]
Hey @ArunTeltia - thanks for checking this page! I can see what you mean about the radio buttons. Currently these are working as we'd expect from a set of radio buttons coded this way, but the exact use case is definitely unique & the interaction via the keyboard isn't intuitive.

If you want to tackle this issue, feel free to create one & start working on it! Looking at the radio buttons, one thing we'll definitely need to do is look at updating their accessible names - right now the `title` attribute is making each radio button have the same name (either "Compare this version..." or "...to this version". Because of the structure, we may need to use `aria-labelledby` attributes.

If you have other issues on your plate (I know you were working on the nav bar stuff) let me know & I'll create the issue myself & we can find someone to work on it!

**Comment by ArunTeltia** [comment]
Definately I will like to work on this 
For the record I researched and got to know that we can also make radiobuttons keyboard accessible by some css work if I remeber correctly if that will be alright I would want to work on it

**Comment by tuminzee** [comment]
@ArunTeltia are you still working on this issue?

**Comment by RayBB** [comment]
Closing as the above feedback makes it clear there's no issue. It's also not a special page, it's our standard wiki and no longer has radio buttons.
